crypto/src/proofs/transparent.rs

```diff
@@ -652,20 +606,17 @@ mod tests {
         let rsk = sk_sender.spend_auth_key().randomize(&spend_auth_randomizer);
         let nk = *sk_sender.nullifier_key();
         let ak = sk_sender.spend_auth_key().into();
-        let mut nct = merkle::BridgeTree::<note::Commitment, 32>::new(5);
-        nct.append(&note_commitment);
-        let anchor = nct.root2();
-        nct.witness();
-        let merkle_path = nct.authentication_path(&note_commitment).unwrap();
+        let mut nct = penumbra_tct::Eternity::new();
```

Comment:
This name is kind of confusing out of context. I think it would be better if the structures from the `tct` crate had `Tree` in the name, like `EternityTree`, `EpochTree`, `BlockTree`, etc.